libs/Microsoft.MixedReality.WebRTC.Native/src/interop/global_factory.cpp

```diff
@@ -21,10 +21,20 @@ using namespace Microsoft::MixedReality::WebRTC;
 /// always valid, never deallocated.
 std::string_view ObjectTypeToString(ObjectType type) {

```

Comment:
nit: a switch might be simpler 😄

Reply:
That's a compile-time switch 😉

Reply:
No it's not! This and a switch are likely to generate the same code.

Reply:
Fixed.